### ISSUE TITLE
Correct sorting of how Messages are displayed in Scheduled tab

### ISSUE THREAD

**Issue by yrimal** [medic/cht-core#8168]
**What feature do you want to improve?**
Outgoing Messages tab in admin section allows users to view messages in different state. While [Due](/admin/#/message-queue/due?page=1), `Will not send`, and `did not send` are sorted in better way so that users see latest messages first, `Scheduled` messages tab is sorted in the opposite way. If you'd like to see the latest Scheduled messages on this tab, you need to navigate 100s of pages.

**Describe the improvement you'd like**
Correct the sorting in Scheduled tab such that we see most recently created Scheduled messages first. It'll be even better if we can configure the sorting order and sorting key. Scheduled messages can be sorted either by Created day or by Scheduled day, but it has to be Descending. 

**Describe alternatives you've considered**
I had to go 995 pages to scee the scheduled messages created today or yesterday. 

**Additional context**
This is the example from one of the instances. 

<img width="610" alt="image" src="https://user-images.githubusercontent.com/6102813/230150069-887cbae4-5dc2-435e-87cb-60fc0eb19ff2.png">

**Comment by dianabarsan** [comment]
The disadvantage to sorting in the opposite direction is that you might end up seeing messages that are supposed to be sent later in the future and have to go to the last pages to see messages that are late.

**Comment by yrimal** [comment]
Yes, but it's still better than going back 6 years, plus always increasing. In new way, maximum you need to go is back from 9 months forward. So, I think it'll still be better use case.

**Comment by garethbowen** [comment]
Why do you have 25k scheduled messages? Is that accurate or because it's a dev instance that'll never actually send anything?

I'm wondering if the scheduled messages should timeout and change status to something terminal. I'm concerned if this instance was ever connected to a gateway for testing or whatever it'd immediately try to send 6 year old messages.

**Comment by yrimal** [comment]
Hi @garethbowen , this is the production instance from last 6 years, currently also in live, and gateway phone connected as well. I observed some interesting things on this tab however. The scheduled messages tab is showing reports from last 6 year only and not after that. 

After last 6 years, you see messages in scheduled tab starting from last 4 months in the middle of page 412 if you look at it now. 

![image](https://user-images.githubusercontent.com/6102813/230389097-ebfe705d-fcc0-4a55-b2c7-fd700459475c.png)

There must have been a config defect 6 years ago and those scheduled were not cleared, I will take that separately in the instance. In all ideal world, scheduled tab shouldn't have messages from the past anyway. I will take it there and come back here to update.

**Comment by garethbowen** [comment]
It's interesting that those 6 year old messages are not actually being sent now - I would expect that everything on that tab will be sent when the gateway is available. This implies to me that the code finds scheduled messages for the tab is inconsistent with the code that actually does the sending.
